common/src/main/java/org/opensearch/ml/common/CommonValue.java

```diff
@@ -30,131 +30,134 @@ public class CommonValue {
 
     public static final String ML_MODEL_INDEX = ".plugins-ml-model";
     public static final String ML_TASK_INDEX = ".plugins-ml-task";
+    public static final String ML_MODEL_RELOAD_INDEX = ".plugins-ml-model-reload";
     public static final Integer ML_MODEL_INDEX_SCHEMA_VERSION = 2;
     public static final Integer ML_TASK_INDEX_SCHEMA_VERSION = 1;
+    public static final String NODE_ID_FIELD = "node_id";
+    public static final String MODEL_LOAD_RETRY_TIMES_FIELD = "retry_times";
     public static final String USER_FIELD_MAPPING = "      \""
-            + CommonValue.USER
-            + "\": {\n"
-            + "        \"type\": \"nested\",\n"
-            + "        \"properties\": {\n"
-            + "          \"name\": {\"type\":\"text\", \"fields\":{\"keyword\":{\"type\":\"keyword\", \"ignore_above\":256}}},\n"
-            + "          \"backend_roles\": {\"type\":\"text\", \"fields\":{\"keyword\":{\"type\":\"keyword\"}}},\n"
-            + "          \"roles\": {\"type\":\"text\", \"fields\":{\"keyword\":{\"type\":\"keyword\"}}},\n"
-            + "          \"custom_attribute_names\": {\"type\":\"text\", \"fields\":{\"keyword\":{\"type\":\"keyword\"}}}\n"
-            + "        }\n"
-            + "      }\n";
+        + CommonValue.USER
+        + "\": {\n"
```

Comment:
These changes are just formatting code. Suggest keep the code format stable, otherwise everyone could change the format, that increase code review effort.